_posts/2015-07-30-decoupling.md

```diff
@@ -9,15 +9,19 @@ smaller repositories. Why? And why should developers and users care?
 
 The OME development workflow has been following a single, integrated approach
 for the last couple of years. This means that all components of the OME stack,
-including OMERO and Bio-Formats are versioned globally and released
-simultaneously. The only notable exception to this rule is the OME Data Model
+including
+[OMERO](https://www.openmicroscopy.org/community/viewtopic.php?f=11&t=7869) and
+[Bio-Formats](https://www.openmicroscopy.org/community/viewtopic.php?f=11&t=7859)
```

Comment:
Did Jason want forum links for OMERO & BF here? I assumed he wanted product page links, we already have the release announcements linked lower down the page.